islandora_xacml_editor.module

```diff
@@ -292,15 +292,16 @@ function islandora_xacml_editor_after_build($form, &$form_state) {
  */
 function islandora_xacml_editor_manage_xacml_form(array $form, array &$form_state, AbstractObject $object) {
   module_load_include('inc', 'islandora', 'includes/utilities');
-  $dt = islandora_basic_collection_get_member_objects($object, 0, 20, 'view', 'islandora:collectionCModel');
-  $child_collections_pids = array();
+  // Hard code the XACML pager element because it needs to be unique.
+  $pager_element = 3;
+  $page = pager_find_page($pager_element);
+  list($count, $results) = islandora_basic_collection_get_member_objects($object, $page, 10, 'manage');
```

Comment:
@jordandukart

I think the final param should have been left intact here `'islandora:collectionCModel'` it filters the list to those objects that can be inherited from.

Thanks!

----
William Panting
_Developer_
*[discoverygarden inc.|http://www.discoverygarden.ca] | Managing Digital Content*